src/features/profile/useProfile.js

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from "react"
+import { useUser } from "../user"
+
+const validate = (value) => {
```

Comment:
**thought:** Should at some point do some schema validation before calling userbase